src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Dto/AttributeForm.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Copyright 2022 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Dto;
+
+class Form
```

Comment:
The class name should reflect the filename. So either call the file `Form.php` or rename the classname to `AttributeForm`

Regardless of that, the name does not sit right with me. This is a value object that aids the attribute type in displaying the label & info popup text. So a name in that direction would make more sense to me. Suggestions: `AttributeTypeInformation`, `AttributeTypeAttributes`, naming things is hard :smile:

Reply:
Agree, AttributeTypeInformation makes more sense.